components/Meta.tsx

```diff
@@ -0,0 +1,33 @@
+import Head from 'next/head'
+
+interface MetaProps {
+  title?: string,
+  keywords?: string,
+  description?: string
+}
+
+const Meta = ({ title, keywords, description }: MetaProps) => {
+  return (
+    <Head>
+      <title>{title}</title>
+      <meta name='keywords' content={keywords} />
+      <meta name="description" content={description} />
+
+      <meta charSet='utf-8' />
+      <meta httpEquiv='X-UA-Compatible' content='IE=edge' />
+      <meta name='viewport' content='width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=5' />
+      <meta name="theme-color" content="#317EFB"/>
+
+      <link rel="icon" href="/favicon.ico" />
+      <link rel="manifest" href="/manifest.json" />
+    </Head>
+  )
+}
+
+Meta.defaultProps = {
+  title: 'Title',
```

Comment:
fixed

Keywords and Description is used in the metadata for chrome. We should keep them; however, I do not know what to add there instead yet.

Reply:
ok gotcha, we can add

`Devfile`, `Registry`, `OpenShift`, `Kubernetes` as some of the keywords
`UI for the Devfile Registry` as the description

this can be done in our next PR